components/analytics-solutions-common/org.wso2.carbon.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateTimePicker.jsx

```diff
@@ -15,9 +15,10 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from "react";
-import {MenuItem, Select} from "material-ui";
-import moment from "moment";
+
+import React from 'react';
+import {MenuItem, SelectField} from 'material-ui';
```

Comment:
Use named imports. Please fix elsewhere

Reply:
This code has been refactored with PR https://github.com/wso2/analytics-solutions/pull/270